crates/bevy_render/src/render_resource/storage_buffer.rs

```diff
@@ -60,14 +64,19 @@ impl<T: ShaderType + WriteInto> StorageBuffer<T> {
         &mut self.value
     }
 
+    /// Debug label of a buffer. This will show up in graphics debuggers for easy identification.
+    pub fn label(&mut self, label: Option<Cow<'static, str>>) {
```

Comment:
IMO this should be `set_label`; we need a seperate `label` getter function here (and below) as the fields aren't public.

Reply:
Is there any reason why label can't just be public? I don't really like trivial setters / getters, but I don't know you do in Rome.

Reply:
Ah, nevermind. If we do your second suggestion the setter would no longer be trivial.

Reply:
My first read is that this could just be public, but I don't know this code super well. I'll defer to @bzm3r and @superdump for their opinion here.

Reply:
If we can get away with it, I'd also prefer it to be pub.